components/dfs/dfs_v2/include/dfs_poll.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
The poll and select implementations are moved to `rt-thread/components/libc/posix/io/poll/select.c`